src/server/webhookServer.ts

```diff
@@ -0,0 +1,74 @@
+import * as http from 'http';
+import _debug from 'debug';
+import { LIBRARY_NAME } from '../constants';
+import { IAlice } from '../alice';
+
+const debug = _debug(LIBRARY_NAME);
+
+export interface IWebhookServerConfig {
+  port: number;
+  webhookUrl: string;
+  options: object;
+  handleRequest: IAlice['handleRequest'];
+}
+export interface IWebhookServer {
+  start(): void;
+  stop(): void;
+}
+
+export class WebhookServer {
+  private server: http.Server;
+  private port: number;
+  private webhookUrl: string;
+  private isServerStarted: boolean;
+  private handleRequest: IAlice['handleRequest'];
+
+  constructor(config: IWebhookServerConfig) {
+    this.port = config.port;
+    this.webhookUrl = config.webhookUrl;
+    this.handleRequest = config.handleRequest;
+    this.isServerStarted = false;
```

Comment:
You dont need word server here. `_isStarted` is enough i think. But it's up to you.

Reply:
hmm, okay, `_isStarted` seems to be better one